js/count-letter-in-string.js

```diff
@@ -0,0 +1,17 @@
+// https://www.codewars.com/kata/52efefcbcdf57161d4000091/train/javascript
+
+function count(string) {
+  const lettersArr = string.split('');
+  let lettersCount = {};
```

Comment:
This could be declared as `const` :smile: But let's not change it, it's good enough

Reply:
Thank you  :)